.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,7 @@ Please mention the type of PR
 
 [ ] Documentation
 
-[ ] Other : | Please Specify |
+[x] Other : | Align center IMGLAB text |
```

Comment:
All the files in *.github* folder are used by github site to display the templates. Please revert any change in this file.